dotcom-rendering/src/server/dev-server.ts

```diff
@@ -38,6 +40,10 @@ export const devServer = (): Handler => {
 				return handleFront(req, res, next);
 			case '/FrontJSON':
 				return handleFrontJson(req, res, next);
+			case '/EmailNewslettersTestData':
+				return handleNewslettersPageWithTestData(req, res, next);
```

Comment:
I'd love to move away from having test data defined in DCR like this, I guess for this to happen https://github.com/guardian/frontend/pull/25876 would need to be merged?